packages/cli/src/commands/governance/conduct-proposal.test.ts

```diff
@@ -0,0 +1,87 @@
+import { newKitFromWeb3 } from '@celo/contractkit'
+import { GovernanceWrapper } from '@celo/contractkit/lib/wrappers/Governance'
+import { testWithGanache, timeTravel } from '@celo/dev-utils/lib/ganache-test'
+
+import BigNumber from 'bignumber.js'
+import fs from 'fs'
+import Web3 from 'web3'
+
+import Register from '../account/register'
+import Lock from '../lockedgold/lock'
+import Approve from './approve'
+import Execute from './execute'
+import Propose from './propose'
+import Vote from './vote'
+
+process.env.NO_SYNCCHECK = 'true'
+const proposalFile = process.env.TEST_PROPOSAL ?? '.tmp/proposal.json'
+
+const DEFAULT_PROPOSAL = [
+  {
+    contract: 'Election',
+    function: 'setElectableValidators',
+    args: [1, 120],
+    value: 0,
+  },
+]
+
+testWithGanache('conduct proposal lifecycle', (web3: Web3) => {
```

Comment:
I see what you did there, but it seems kind of odd to use a test file as an invocation against a network? What do you think about extracting a library method that gets called in the test and in a specific script?

Reply:
definitely odd lol
I figured we could beef up the cli coverage simultaneously without having to dupe a bunch of logic to protocol package

Reply:
I think you can do that with just adding a CLI command as well? In any case, up to you

Reply:
fwiw the new test caught a breaking change to contractkit that would have bricked all the governance commands
see the `ERC20` change

Reply:
after some more thought, anything other than a ganache network will not work well with a similar "library method" because of the quite long timing dependency
unless we have a `sleep 3 days` or something

Reply:
For sure only applies to dev networks, but wouldn't it still be more ergonomic/intuitive to have a specific command/script rather than executing a test suite with ENV vars? I think you can still get the benefits of coverage with a shared library?

Reply:
I don't love the idea of distributing a command in celocli which only works against dev networks
A shared library makes sense.